app/src/main/res/values/strings.xml

```diff
@@ -649,10 +649,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="theme_light">Light</string>
     <string name="theme_dark">Dark</string>
     <string name="theme_system_default">System default</string>
+    <string name="pref_title_overpass_url_select">Change Overpass server</string>
+    <string name="pref_title_overpass_url_info">may bypass censorship, for example in Russia</string>
     <string name="quest_accessible_for_pedestrians_separate_sidewalk_explanation">This street was tagged as having no sidewalk on either side. In the case that there is a sidewalk after all but it is displayed as a separate way, please answer \"sidewalk\".</string>
     <string name="action_open_location">Open location in another app</string>
     <string name="map_application_missing">No other map application installed</string>
     <string name="label_housenumber">house number</string>
     <string name="label_blocknumber">block number</string>
     <string name="quest_busStopShelter_covered">Whole stop is covered</string>
+    <string name="overpass_server_default">Default overpass server provided by Roland Olbricht</string>
```

Comment:
The server is provided/sponsored by FOSSGIS e.V.

Reply:
What version of attribution would be best?

> Default overpass server provided by FOSSGIS, administered by Roland Olbricht

or 

> Default overpass server provided by FOSSGIS e.V., administered by Roland Olbricht

or

> Default overpass server provided by FOSSGIS e.V.


Pinging also @drolbr

Reply:
moot as result of 

> Actually, I would very much prefer not to add a description of the Overpass URL to the localized strings but simply display the URL.

request